gnmi_server/client_subscribe.go

```diff
@@ -197,7 +197,7 @@ func (c *Client) recv(stream gnmipb.GNMI_SubscribeServer) {
 		switch err {
 		default:
 			log.V(1).Infof("Client %s received error: %v", c, err)
-			return
+			break
```

Comment:
>break [](http://example.com/codeflow?start=3&length=5)

This modification changes behavior.

If you are fixing "This statement is unreachable." then just remove the unreachable code.